code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/common/AEPPresentable.kt

```diff
@@ -315,6 +319,8 @@ internal abstract class AEPPresentable<T : Presentation<T>> :
             return
         }
 
+        activityCompatOwnerUtils.attachActivityCompatOwner(activityToAttach)
```

Comment:
[optional] Should we update the `attach()`'s java docs for the `attach`/`detach` changes?